README.md

```diff
@@ -105,6 +105,37 @@ work         user@mycorp.org
 By default, the Dropbox folder names will contain the capitalised config-name in braces.
 In the above case, this will be "Dropbox (Personal)" and "Dropbox (Work)".
 
+### Docker usage
+
+The Docker image is available for x86, arm/v7 (32bit) and arm64 platforms. You can do
+everything that you supposed to do in the command line, except running the GUI.
```

Comment:
Yeah, that's a great idea, but that would require to me to do some digging into the topic, setting up a VM with a DBus, because I use Maestral on my NAS that has no GUI. :) I see DBus errors in the log, but because they are not excessive, I just ignored them. The syncing on a 1.5TB Dropbox account worked like a charm, especially after I tweaked the `inotify` settings with `sysctl`. :)

For DBus and Keyring, I'd recommend to create a separate GH issue for each as they are separate problems and Maestral on my Raspberry Pi OSMC and on an Ubuntu one with ZFS works like a charm. (Thanks again for your hard work!)

Reply:
I am glad its working well for you!

I have no idea how difficult connecting to a host's dbus session will be. A google search brings up all kinds of solutions which people have tried with varying success so I am willing to push this into a separate GH issue.